examples/basic/src/experimental/guard/guard-retry-fix-structure-add-error-to-prompt-example.ts

```diff
@@ -25,7 +25,7 @@ async function main() {
         zodSchema(
           z.object({
             sentiment: z
-              .enum(["positivee", "neutra", "negaaa"])
+              .enum(["positive", "neutral", "negative"])
```

Comment:
The mispellings are intentional here - pls revert

Reply:
done.